src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM);
```

Comment:
There's a deprecation notice at [OperationTypeDeprecationHelper.php#L39](https://github.com/api-platform/core/blob/42940e3f48c3450d6c193f6bca6b8c5041821ffa/src/Api/OperationTypeDeprecationHelper.php#L39) that suggests that although the boolean type works at the moment it will stop working in v3. Happy to revert the change if it is premature.